awx/ui_next/src/screens/User/shared/UserTokenForm.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Formik, useField } from 'formik';
+import { Form, FormGroup } from '@patternfly/react-core';
+import AnsibleSelect from '../../../components/AnsibleSelect';
+import FormActionGroup from '../../../components/FormActionGroup/FormActionGroup';
+import FormField, {
+  FormSubmitError,
+  FieldTooltip,
+} from '../../../components/FormField';
+import ApplicationLookup from '../../../components/Lookup/ApplicationLookup';
+import { required } from '../../../util/validators';
+
+import { FormColumnLayout } from '../../../components/FormLayout';
+
+function UserTokenFormFields({ i18n }) {
+  const [applicationField, applicationMeta, applicationHelpers] = useField(
+    'application'
+  );
+
+  const [scopeField, scopeMeta, scopeHelpers] = useField({
+    name: 'scope',
+    validate: required(i18n._(t`Please enter a value.`), i18n),
+  });
+
+  return (
+    <>
+      <FormGroup
+        fieldId="application-lookup"
+        name="application"
+        validated={
+          !applicationMeta.touched || !applicationMeta.error
+            ? 'default'
+            : 'error'
+        }
+        helperTextInvalid={applicationMeta.error}
+      >
+        <ApplicationLookup
+          value={applicationField.value}
+          onChange={value => {
+            applicationHelpers.setValue(value);
+          }}
+          label={
+            <span>
+              {i18n._(t`Application`)}
+              <FieldTooltip
+                content={i18n._(
+                  t`Select the application that this token will belong to.`
+                )}
+              />
+            </span>
+          }
+          touched={applicationMeta.touched}
+        />
+      </FormGroup>
+      <FormField
+        id="token-description"
+        name="description"
+        type="text"
+        label={i18n._(t`Description`)}
+      />
+
+      <FormGroup
+        name="scope"
+        fieldId="token-scope"
+        helperTextInvalid={scopeMeta.error}
+        validated={!scopeMeta.touched || !scopeMeta.error ? 'default' : 'error'}
+        label={
+          <span>
+            {i18n._(t`Scope`)}
+            <span className="pf-c-form__label-required" aria-hidden="true">
+              *
+            </span>
+            <FieldTooltip
+              content={i18n._(t`Specify a scope for the token's access`)}
+            />
+          </span>
+        }
+      >
```

Comment:
FormGroup has an `isRequired` prop you can set to add the red asterisk, so you don't have to do it by hand here

Reply:
@keithjgrant I know about the `isRequired` prop. However, since the PF4 upgrade the alignment of the labels has been off.  See https://github.com/ansible/awx/issues/7556.  This is an attempt to solve that problem.  IMO for this specific use case we could go with out established pattern and fix this case when we fix the others (which I'm happy to do), or leave it this way and use it as a model the solve/starting place to solve the others.